src/components/primary-toolbar/primary-toolbar.test.js

```diff
@@ -62,7 +57,6 @@ describe('PrimaryToolbar', () => {
     const expectedResult = {
       disableLayerBtn: expect.any(Boolean),
       textLabels: expect.any(Boolean),
-      theme: expect.stringMatching(/light|dark/),
       visible: expect.objectContaining({
         exportBtn: expect.any(Boolean),
         settingsBtn: expect.any(Boolean),
```

Comment:
This can be deleted - same for `themeBtn` for line 68 as well

Reply:
It's gone now.